doc/source/contribute/developer.rst

```diff
@@ -0,0 +1,182 @@
+Contributing as a developer
+###########################
+
+.. grid:: 1 1 3 3
+
+    .. grid-item-card:: :fa:`code-fork` Fork the repository
+        :padding: 2 2 2 2
+        :link: fork-the-repository
+        :link-type: ref
+
+        Learn how to fork the project and get your own copy.
+
+    .. grid-item-card:: :fa:`download` Clone the repository
+        :padding: 2 2 2 2
+        :link: clone-the-repository
+        :link-type: ref
+
+        Download your own copy in your local machine.
+
+    .. grid-item-card:: :fa:`download` Install for developers
+        :padding: 2 2 2 2
+        :link: install-for-developers
+        :link-type: ref
+
+        Install the project in editable mode.
+
+    .. grid-item-card:: :fa:`vial-circle-check` Run the tests
+        :padding: 2 2 2 2
+        :link: run-tests
+        :link-type: ref
+
+        Verify your changes by testing the project.
+
+
+.. _fork-the-repository:
+
+Fork the repository
+===================
+
+Forking the repository is the first step to contributing to the project. This
+allows you to have your own copy of the project so you can make changes without
+affection the main project. Once you have made your changes, you can submit a
+pull-request to the main project to have your changes reviewed and merged.
+
+.. button-link:: https://github.com/ansys/pyworkbench/fork
+    :color: primary
+    :align: center
+
+    :fa:`code-fork` Fork this project
+
+.. note::
+
+    If you are an Ansys employee, you can skip this step.
```

Comment:
What about the instructions for external contributors?

Reply:
@jorgepiloto I can see now that my comment doesn't really make sense. The "Fork the repository" step doesn't have to be completed by Ansys employees, and external contributors can refer to GitHub resources or the PyAnsys dev guide for forking instructions.  Should we say something to this effect though--or is it just obvious to developers?